GOVERNANCE.rst

```diff
@@ -335,16 +346,51 @@ sktime uses a “consensus seeking” process for making decisions. The
 community tries to find a resolution that has no open objections among
 core developers.
 
--  To accept proposed changes, we require approval by one core developer
-   (lazy consensus) and no rejection by a core developer (veto right).
+-  For a proposed changes to be approved via lazy consensus, one of the following
+   two conditions must be satisfied:
+    - approval by at least one core developer and no rejection by a core developer (veto right).
+      The approval required for this condition must be by a core developer different from a proposer.
+    - approval in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
```

Comment:
Let's rely on explicit approvals on GitHub
```suggestion
```
Also makes it easier: you need one approval (lazy consensus) and no rejection (veto right)

Reply:
but that's still an option! I'm just spelling out the approval conditions. Either what you say (approval, no veto), or approval in the core dev meeting if neither approval nor veto appears on the PR.